File: js/delivery-api/GetTaxonomyGroups.js
// DocSection: delivery_api_get_taxonomy_groups
// Tip: Find more about JS/TS SDKs at https://kontent.ai/learn/javascript
const KontentDelivery = require('@kontent-ai/delivery-sdk');

const deliveryClient = KontentDelivery.createDeliveryClient({
  environmentId: '<YOUR_ENVIRONMENT_ID>'
});

const response = await deliveryClient.taxonomies()
  .limitParameter(3)
  .toPromise();
// EndDocSection

File: java/delivery-api/GetTaxonomyGroups.java
// DocSection: delivery_api_get_taxonomy_groups
// Tip: Find more about Java SDK at https://docs.kontent.ai/java
import kentico.kontent.delivery.*;

DeliveryClient client = new DeliveryClient("<YOUR_PROJECT_ID>");

List<NameValuePair> params = DeliveryParameterBuilder.params().page(null, 3).build();

CompletionStage<TaxonomyGroupListingResponse> response = client.getTaxonomyGroups(params);
// EndDocSection